### PR TITLE
doc cli lint.

### PR DESCRIPTION
**Pull request by dwlf**
- 79 char line length (non-command).
- Consistent single space between sentences.
- Replace start of line tabs with spaces.
- Remove trailing white space.
- Consistent "**Note:**" style.
- Replace the one ``` section with indented code.

Signed-off-by: Lloyd Dewolf foolswisdom@gmail.com

### CODE REVIEW

**Comment by thaJeztah** [comment]
Oh cool! I'm about to go into a meeting, but will definitely look at it after that. We should try to get this merged soon, because it's a document that is touched frequently.

ping @fredlf @jamtur01 @moxiegirl @SvenDowideit

**Comment by jamtur01** [comment]
Broadly LGTM - nice work!

**Comment by lloydde** [comment]
@thaJeztah no rush, I can easily rebase as needed.

File: docs/sources/reference/commandline/cli.md [hunk 1]
```diff
@@ -197,30 +201,31 @@ To run the daemon with debug output, use `docker -d -D`.
...
+Socket. Beware that the default setup provides un-encrypted and
+un-authenticated direct access to the Docker daemon - and should be secured
+either using the [built in HTTPS encrypted socket](/articles/https/), or by
+putting a secure web proxy in front of it. You can listen on port `2375` on all
+network interfaces with `-H tcp://0.0.0.0:2375`, or on a particular network
+interface using its IP address: `-H tcp://192.168.59.103:2375`. It is
+conventional to use port `2375` for un-encrypted, and port `2376` for encrypted
+communication with the daemon.
+
+> **Note** If you're using an HTTPS encrypted socket, keep in mind that only
```
Comment (thaJeztah) [inline docs/sources/reference/commandline/cli.md]: Missing a colon (`:`) here, after the "Note"

File: docs/sources/reference/commandline/cli.md [hunk 2]
```diff
@@ -463,8 +471,8 @@ Currently supported options of `zfs`:
 
        $ docker -d -s zfs --storage-opt zfs.fsname=zroot/docker
```
Comment (thaJeztah) [inline docs/sources/reference/commandline/cli.md]: Looks like this line isn't rendered as code-block. Possibly needs more indenting

(Not introduced by your changes)

File: docs/sources/reference/commandline/cli.md (hunk 1 above)
Comment (lloydde) [inline docs/sources/reference/commandline/cli.md]: Do we want the colon in the bold? I think yes, but just now noticed that is inconsistent.

**Comment by thaJeztah** [comment]
Like @jamtur01, generally LGTM. I checked the source diff and didn't spot problems, and checked visually on GitHub and all looked well

I added two small notes.

I would like one extra pair of eyes on this, because it's a large diff and I might have missed something

Great job!  Did you use some tooling for this?

File: docs/sources/reference/commandline/cli.md (hunk 1 above)
Comment (thaJeztah) [inline docs/sources/reference/commandline/cli.md]: Erm, yes in bold, see https://docs.docker.com/project/doc-style/#notes

But I forget all the time :)

File: docs/sources/reference/commandline/cli.md [hunk 3]
```diff
@@ -56,18 +58,18 @@ The Docker command line stores its configuration files in a directory called
...
 
 The `config.json` file stores a JSON encoding of a single `HttpHeaders`
-property. The property specifies a set of headers to include in all
-messages sent from the Docker client to the daemon. Docker does not try to
-interpret or understand these header; it simply puts them into the messages.
-Docker does not allow these headers to change any headers it sets for itself.
+property. The property specifies a set of headers to include in all messages
+sent from the Docker client to the daemon. Docker does not try to interpret or
+understand these header; it simply puts them into the messages. Docker does
+not allow these headers to change any headers it sets for itself.
```
Comment (moxiegirl) [inline docs/sources/reference/commandline/cli.md]: The antecedent it a bit unclear.  Maybe just:

"The headers that Docker sets for itself cannot be changed."

File: docs/sources/reference/commandline/cli.md [hunk 4]
```diff
@@ -98,16 +102,16 @@ you can write `docker run -it --name test busybox sh`.
 
 ### Boolean
 
-Boolean options take the form `-d=false`. The value you see in the help text is the
-default value which is set if you do **not** specify that flag. If you specify
-a Boolean flag without a value, this will set the flag to `true`, irrespective
-of the default value.
+Boolean options take the form `-d=false`. The value you see in the help text is
+the default value which is set if you do **not** specify that flag. If you
```
Comment (moxiegirl) [inline docs/sources/reference/commandline/cli.md]: We can clarify this by using a restrictive clause and tighten the structure: ->

The value displayed in the help is the default Docker uses if you omit that flag. If you specify a Boolean flag without a value, Docker uses a value of `true` irrespective of the flag's default.

File: docs/sources/reference/commandline/cli.md [hunk 5]
```diff
@@ -98,16 +102,16 @@ you can write `docker run -it --name test busybox sh`.
...
+specify a Boolean flag without a value, this will set the flag to `true`,
+irrespective of the default value.
 
-For example, running `docker run -d` will set the value to `true`, so
-your container **will** run in "detached" mode, in the background.
+For example, running `docker run -d` will set the value to `true`, so your
```
Comment (moxiegirl) [inline docs/sources/reference/commandline/cli.md]: image

Also, don't need quotes around detached here and no. so --->

For example, the `docker run-d` command causes Docker to use `true` and your container then runs in the detached mode.

File: docs/sources/reference/commandline/cli.md [hunk 6]
```diff
@@ -98,16 +102,16 @@ you can write `docker run -it --name test busybox sh`.
...
+container **will** run in "detached" mode, in the background.
 
-Options which default to `true` (e.g., `docker build --rm=true`) can only
-be set to the non-default value by explicitly setting them to `false`:
+Options which default to `true` (e.g., `docker build --rm=true`) can only be
```
Comment (moxiegirl) [inline docs/sources/reference/commandline/cli.md]: which > that 
inconsistent use of flag or option (not just use but maybe we can clean some up) -->

When a flag defaults to `true`, you can only set it to `false` by explicitly passing the flag:

**Comment by lloydde** [comment]
@moxiegirl my changes are only in formatting of the file not in correcting the content. Could those issues be addressed in a separate github issue?

**Comment by lloydde** [comment]
@thaJeztah I've integrated your feedback. Let me know if/when to squash the commits.

**Comment by lloydde** [comment]
> Did you use some tooling for this?
@thaJeztah

Not as automated as I would like it to be, but I've starting to build up my vim plugins to assist in markdown editing.

Somewhat related, for more one purposes, I'm thinking of switching linkings to implicit as it is helping me with reviewing prose:

```
I get 10 times more traffic from [Google][] than from
[Yahoo][] or [MSN][].

  [google]: http://google.com/        "Google"
  [yahoo]:  http://search.yahoo.com/  "Yahoo Search"
  [msn]:    http://search.msn.com/    "MSN Search"
```

**Comment by moxiegirl** [comment]
@lloydde yep. I just realized you hadn't actually changed the text. Ignore my comments. Thank you for the contribution. 
LGTM

**Comment by thaJeztah** [comment]
@lloydde 

> Somewhat related, for my one purpose I'm thinking of switching linkings to implicit as it is helping me with reviewing prose

I like that way of links in Markdown, but it could make it harder to maintain and, when reading the markdown in plain text, harder to find the links. So I think it's better kept for a separate discussion.

**Comment by thaJeztah** [comment]
Keep us posted on how you progress on automating this! It's still risky to fully automate, but interesting still.

If you are interested in addressing @moxiegirl's suggestions as a follow-up that would be great! But don't feel obliged to do so.

I looked at your last commit and LGTM! Could you squash your commits, then I think we should merge.

**Comment by lloydde** [comment]
@thaJeztah ready for final review and merging. Thank you.

**Comment by thaJeztah** [comment]
Excellent! Thanks so much for this.

Merging :)

**Pull request merged**